### DISCUSSION TITLE
Icons for LSP popup

### DISCUSSION THREAD

**Discussion by KittyBorgX** [helix-editor/helix#7103] [Q&A] [answered]
Is there any way to add icons to the LSP autocomplete popup menu? 
Currently it looks like this: 
![image](https://github.com/helix-editor/helix/assets/82328083/0c438fae-6f1c-4e1c-b883-a8eafd805482)

But I want it to look something more like vscode, like this: 
![image](https://github.com/helix-editor/helix/assets/82328083/77beec03-8e39-445d-91ce-671c1df7f3e7)

**Comment by CptPotato** [comment, marked as answer]
There currently is no option to do this. There's a relevant PR though: #2869

**Reply by KittyBorgX** [reply]
Alright! Thank you so much!
